### PR TITLE
Use cards for claim/receive accounts.

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Styles the claim/receive accounts
- For now keeping the Badge colors provided by the component @anxolin  added.

Screen Shot 2022-01-26 at 18 32 24
Screen Shot 2022-01-26 at 18 32 16

### CODE REVIEW

**Comment by biocom** [comment]
Just noticed the icon color in light mode. Will address.

**Comment by biocom** [comment]
Fix pushed for the icon color in light mode:
Screen Shot 2022-01-26 at 18 54 18

File: src/custom/pages/Claim/InvestmentFlow/index.tsx [hunk 1]
```diff
@@ -102,27 +103,32 @@ function _calculateTotalVCow(allClaims: ClaimWithInvestmentData[]) {
 }
 
 type AccountDetailsProps = {
+  index: number
```
Comment (alfetopito) [inline src/custom/pages/Claim/InvestmentFlow/index.tsx:106]: May I suggest, to be more semantic:

```suggestion
  isClaimer?: boolean
```

File: src/custom/pages/Claim/InvestmentFlow/index.tsx [hunk 2]
```diff
@@ -102,27 +103,32 @@ function _calculateTotalVCow(allClaims: ClaimWithInvestmentData[]) {
...
+function AccountDetails({ index, label, account, connectedAccount }: AccountDetailsProps) {
   return (
-    <span>
-      <b>
-        <Icon width={14} height={14} /> {label}:
-      </b>
-      <i>
-        <ExplorerLink id={account} label={account} type={ExplorerDataType.ADDRESS} />{' '}
+    <div>
+      {index === 0 && (
```
Comment (alfetopito) [inline src/custom/pages/Claim/InvestmentFlow/index.tsx:115]: Using the suggestion above, here we'd have:

```suggestion
      {isClaimer && (
```

File: src/custom/pages/Claim/InvestmentFlow/index.tsx [hunk 3]
```diff
@@ -245,17 +251,12 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, modalCbs }: I
           </ClaimTable>
 
           <AccountClaimSummary>
+            <AccountDetails index={0} label="Claiming with account" account={account} connectedAccount={account} />
```
Comment (alfetopito) [inline src/custom/pages/Claim/InvestmentFlow/index.tsx:254]: And you would only need to include the flag in the first component

```suggestion
            <AccountDetails isClaimer label="Claiming with account" account={account} connectedAccount={account} />
```

**Review by alfetopito — APPROVED** [review]
How can you make it look so good?

Approved, just small suggestion regarding the semantics of a prop

**Comment by elena-zh** [comment]
@biocom , magic you do!
But could we make addresses arrows clickable?
image

Also, could we show these addresses in 1 line in a mobile view? 
image

Thanks!

File: src/custom/pages/Claim/InvestmentFlow/index.tsx (hunk 3 above)
Comment (biocom) [reply src/custom/pages/Claim/InvestmentFlow/index.tsx:254]: @alfetopito Aaah much cleaner like that, thank you!

File: src/custom/pages/Claim/InvestmentFlow/index.tsx [hunk 4]
```diff
@@ -245,17 +251,12 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, modalCbs }: I
           </ClaimTable>
 
           <AccountClaimSummary>
+            <AccountDetails isClaimer label="Claiming with account" account={account} connectedAccount={account} />
```
Comment (anxolin) [inline src/custom/pages/Claim/InvestmentFlow/index.tsx:254]: I thought this a few times, would it be more clear to say `Investing account` or `Paying account` to highlight who is the one incurring to costs and which one benefits

**Review by anxolin — APPROVED** [review]
You are a 🧙‍♀️

**Comment by biocom** [comment]
@elena-zh Thanks for reporting. Pushed a fix for both items:
Screen Shot 2022-01-26 at 21 16 38

Mobile
Screen Shot 2022-01-26 at 21 15 59

File: src/custom/pages/Claim/InvestmentFlow/index.tsx (hunk 4 above)
Comment (biocom) [reply src/custom/pages/Claim/InvestmentFlow/index.tsx:254]: Two things could be happening (at the same time or not):
- You are **claiming** an airdrop.
- You are **[buying||investing in]** vCOW tokens.

Also the connected account is the account **paying** for both of the above.

So not sure what word we should prioritize here, in order to be as consistent as possible. I can image something like **'Executing account'** covering all and being more agnostic. But feel free to change wording here.

**Comment by biocom** [comment]
Merging to keep release up to date.
@anxolin Feel free to modify wording as you see fit, based on the comment thread in this PR
@elena-zh please review the 2 points I addressed

File: src/custom/pages/Claim/InvestmentFlow/index.tsx (hunk 4 above)
Comment (alfetopito) [reply src/custom/pages/Claim/InvestmentFlow/index.tsx:254]: What about one word shorter:

```suggestion
            <AccountDetails isClaimer label="Claiming with" account={account} connectedAccount={account} />
```

File: src/custom/pages/Claim/InvestmentFlow/index.tsx [hunk 5]
```diff
@@ -244,18 +251,8 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, modalCbs }: I
...
-              Icon={DollarSign}
-            />
-            <AccountDetails
-              label="Receiving account"
-              account={activeClaimAccount}
-              connectedAccount={account}
-              Icon={Send}
-            />
+            <AccountDetails isClaimer label="Claiming with account" account={account} connectedAccount={account} />
+            <AccountDetails label="Account receiving vCOW" account={activeClaimAccount} connectedAccount={account} />
```
Comment (alfetopito) [inline src/custom/pages/Claim/InvestmentFlow/index.tsx:255]: And here also a bit shorter:

```suggestion
            <AccountDetails label="Receiving on" account={activeClaimAccount} connectedAccount={account} />
```

The we'll have:

Claiming with 0x000...
Receiving on 0x000...

Comment (biocom) [reply src/custom/pages/Claim/InvestmentFlow/index.tsx:255]: Yes, good with me 👍🏼

**Comment by elena-zh** [comment]
Hey @biocom , looks great now!
Tiny-tiny nitpick is that navigation arrow has a bit differently underlined than an address in Win10+Chrome
image

As an expected result I'd prefer it underlined the same way as we do for 'View all orders' link or so 
image

But anyways, if you think that this nitpick worth effort, it will be addressed in another PR. 

Thanks!

**Pull request merged**